modules/sdk-coin-avaxp/src/lib/transactionBuilder.ts

```diff
@@ -132,7 +99,72 @@ export abstract class TransactionBuilder extends BaseTransactionBuilder {
    */
   protected abstract buildAvaxpTransaction(): void;
 
+  // endregion
+
   // region Getters and Setters
+
+  protected get outsMethod(): string {
+    return 'getOuts';
+  }
+
+  protected get insMethod(): string {
+    return 'getIns';
+  }
+
+  /**
+   * When using recovery key must be set here
+   * TODO: STLX-17317 recovery key signing
+   * @param recoverSigner
+   */
+  public recoverMode(recoverSigner = true): this {
+    this.recoverSigner = recoverSigner;
+    return this;
+  }
+
+  threshold(value: number): this {
+    this.validateThreshold(value);
+    this._transaction._threshold = value;
+    return this;
+  }
+
+  locktime(value: string | number): this {
```

Comment:
for instance, is the value in seconds? microseconds? nano?
is it okay to have decimals?